plugins/aggregators/merge/sample.conf

```diff
@@ -6,6 +6,21 @@
   ## is also rounded.
   # round_timestamp_to = "1ns"
 
+  ## The period on which to flush & clear each aggregator. 
+  ## All metrics that are sent with timestamps outside of this period will be ignored by the aggregator.
+  # period = "30s"
+
+  ## The delay before each aggregator is flushed. 
+  ## This is to control how long for aggregators to wait before receiving metrics from input plugins, 
+  ## in the case that aggregators are flushing and inputs are gathering on the same interval.
+  # delay = "100ms"
+
+  ## The duration when the metrics will still be aggregated by the plugin, 
+  ## even though they're outside of the aggregation period. 
+  ## This is needed in a situation when the agent is expected to receive late metrics 
+  ## and it's acceptable to roll them up into next aggregation period.  
+  # grace = "0s"
+
```

Comment:
I don't think we should add this, it's covered by the `Global configuration options` section of the README... Otherwise we would also need to add all tag/field include/exclude options etc... What do you think?